x-pack/plugins/cases/public/common/translations.ts

```diff
@@ -232,6 +232,12 @@ export const SELECTABLE_MESSAGE_COLLECTIONS = i18n.translate(
 export const SELECT_CASE_TITLE = i18n.translate('xpack.cases.common.allCases.caseModal.title', {
   defaultMessage: 'Select case',
 });
+export const CANCEL_ADD_TO_CASE_BUTTON = i18n.translate(
```

Comment:
Hi! There is already a `cancel` word here: https://github.com/claudiopro/kibana/blob/main/x-pack/plugins/cases/public/common/translations.ts#L14 not tied to any component that you can use.

Reply:
Cool, I didn't reuse the existing string in order to make the button label resilient to changes to other references, but I see this string is widely used across many **Cases** related components.

Reply:
I got that 🙂